str/associatr/helper/snp_vcf_for_associatr.py

```diff
@@ -64,8 +64,16 @@ def reformat_vcf(vcf_file_path, output_file_path):
                 pid_index = format_field.split(':').index('PID') if 'PID' in format_field.split(':') else None
                 pid = sample_data[0].split(':')[pid_index] if pid_index is not None else '.'
 
-                # Update INFO field (add new fields required by ExpansionHunter, filler fields eg REF and RU are a constant 3)
-                new_info_fields = [f'END={parts[1]}', 'REF=3', f'REPID={pid}', 'RL=0', 'RU=3', f'VARID={pid}']
+                # Update INFO field (add new fields required by ExpansionHunter, filler fields eg REF =0)
+                # RU stores the ref and alt_allele; this will allow us to discriminate between loci with the same POS
+                new_info_fields = [
+                    f'END={parts[1]}',
+                    'REF=3',
+                    f'REPID={pid}',
+                    'RL=0',
+                    f'RU={parts[3]}-{parts[4]}',
```

Comment:
Worried about populating a field with misleading information and not having any way of going back and deciphering.
Can we change the header information on lines #L44 to something like this?
```
fout.write('##INFO=<ID=RU,Number=1,Type=String,Description="Storing the REF/ALT info instead of Repeat Unit to retain the REF/ALT info in the association output files, which is crucial where there are multiple variants with the same CHR:POS coordinates (eg multi allelic loci)">\n')